credentials/alts/alts.go

```diff
@@ -91,6 +91,19 @@ type AuthInfo interface {
 	PeerRPCVersions() *altspb.RpcProtocolVersions
 }
 
+// ClientOptions contains the client-side options of an ALTS channel. These
+// options will be passed to the underlying ALTS handshaker.
+type ClientOptions struct {
+	// TargetServiceAccounts contains a list of expected target service
+	// accounts.
+	TargetServiceAccounts []string
+}
+
+// DefaultClientOptions creates a default (empty) ClientOptions.
+func DefaultClientOptions() *ClientOptions {
```

Comment:
This should not be necessary; your zero-value defaults should all be sane (and I see you do end up with the zero value, so this is true).

Reply:
Done!